ocaml/xapi/create_misc.ml

```diff
@@ -387,7 +387,38 @@ let make_packs_info () =
 		in
 		Array.fold_left (fun l fname -> get_pack_details fname @ l) [] packs
 	with _ -> []
-  
+
+let trim s = String.filter_chars s (not ++ String.isspace)
```

Comment:
"Trim" normally means "remove whitespace from the beginning and end of the string", but this function removes _all_ whitespace. Is that what you want? A conventional "trim" function is available in the standard lib (`String.trim`).